src/components/AccessibilityPolicyForm/AccessibilityPolicyForm.test.jsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import { AccessibilityPolicyForm } from './index';
+import { accessibilityActions } from '../../data/constants/actionTypes';
+
+const mount = Enzyme.mount;
+
+const defaultProps = {
+  accessibilityStatus: {},
+  email: 'accessibilityTest@test.com',
+  clearAccessibilityStatus: () => {},
+  submitAccessibilityForm: () => {},
+};
+
+const formInputs = {
+  email: 'test@test.com',
+  fullName: 'test name',
+  message: 'feedback message',
+};
+
+const validationMessages = {
+  email: 'Valid email required.',
+  fullName: 'Full name required.',
+  message: 'Message required.',
+};
+
+const clearStatus = (wrapper) => {
+  wrapper.setProps({ accessibilityStatus: {} });
+};
+
+const getMockForZendeskSuccess = (wrapper) => {
+  wrapper.setProps({
+    accessibilityStatus: {
+      type: accessibilityActions.ACCESSIBILITY_FORM_SUBMIT_SUCCESS,
+    },
+  });
+};
+
+const getMockForZendeskRateLimit = (wrapper) => {
+  wrapper.setProps({
+    accessibilityStatus: {
+      type: accessibilityActions.ACCESSIBILITY_FORM_SUBMIT_RATE_LIMIT_FAILURE,
+    },
+  });
+};
+
+let wrapper;
+
+describe('<AccessibilityBody />', () => {
+  describe('renders', () => {
+    beforeEach(() => {
+      wrapper = mount(
+        <AccessibilityPolicyForm
+          {...defaultProps}
+        />,
+      );
+    });
+
+    it('correct number of form fields', () => {
+      const formSection = wrapper.find('section');
+      expect(formSection.find('input')).toHaveLength(2);
+      expect(formSection.find('textarea')).toHaveLength(1);
+      expect(formSection.find('button')).toHaveLength(1);
+    });
+
+    it('hides StatusAlert on initial load', () => {
+      const statusAlert = wrapper.find('StatusAlert');
+      expect(statusAlert.find('div').first().prop('hidden')).toEqual(true);
+    });
+
+    it('adds validation checking on each input field', () => {
+      const emailInput = wrapper.find('input#email');
+      emailInput.simulate('blur');
+      // expect(spy).toHaveBeenCalledTimes(1);
```

Comment:
Do you need this line? If not, you should remove it.

Reply:
There's a few commented lines in this file.

Reply:
That can be removed, it was mostly to remind me to ask someone on the team how to properly leverage mock functions in tests :) I feel like I've mostly found work arounds for doing so for right now and want to to see if I could be writing these tests in a better way.

Reply:
Does jest.fn() not work as usual?

Reply:
It probably works as it usually has, but it wasn't working as I was expecting, so I'm guessing that I was making some weird assumptions

Reply:
If it's something that you'd like to tackle this sprint and would like another eye, I'm happy to take a look, if I can be helpful. If not, can we make a note of it somewhere so we can make sure to resolve it?